devito/operator/operator.py

```diff
@@ -635,7 +635,7 @@ def cfunction(self):
     # Execution
 
     def __call__(self, **kwargs):
-        self.apply(**kwargs)
+        return self.apply(**kwargs)
```

Comment:
?

Reply:
`self.apply` returns the summary ! we've had this bug for like forver -- when using the `op()` shortcut we were systematically returning None instead of the Summary dict. I bumped into it accidentally while writing the test

Reply:
Oooooh good catch